paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountForm.kt

```diff
@@ -287,7 +300,11 @@ internal fun BillingDetailsForm(
 ) {
     Column(Modifier.fillMaxWidth()) {
         H6Text(
-            text = stringResource(R.string.stripe_paymentsheet_pay_with_bank_title),
+            text = if (isPaymentFlow) {
```

Comment:
We really need to improve testing here. Is there a test you can add for at least this case?

Reply:
I will do this in a follow up PR